user/Kconfig

```diff
@@ -1,11 +1,18 @@
-# Copyright (c) 2015 The F9 Microkernel Project. All rights reserved.
+# Copyright (c) 2015-2016 The F9 Microkernel Project. All rights reserved.
 # Use of this source code is governed by a BSD-style license that can be
 # found in the LICENSE file.
 
-menu "Test Case"
-config EXTI_INTERRUPT_TEST
-	bool "EXTI interrupt test case"
+menu "User Space"
+
+config USER_APPS_INCLUDE
```

Comment:
I dislike the use of `USER_APPS_INCLUDE`. The mnemonic alternative can be `USER_APPS_ENABLED`.